pkg/configmapandsecret/configmapfactory.go

```diff
@@ -70,6 +71,7 @@ func objectToUnstructured(in runtime.Object) (*unstructured.Unstructured, error)
 	}
 	var out unstructured.Unstructured
 	err = out.UnmarshalJSON(marshaled)
+	out.SetCreationTimestamp(metav1.Time{})
```

Comment:
`objectToUnstructured` might not be the best place to add the line of setting the creationTimestamp. Can you move this line to functions `MakeConfigMap` and `MakeSecret`?